### PR TITLE
Shilpi newcastle brca training

### PR DESCRIPTION
**Pull request by shilpigoeldev**
I have completed changes for Newcastle BRCA , adhering to the final rules and Francesco has signed off QA .

### CODE REVIEW

File: lib/import/brca/scripts/bash/Import_all_brca_interactive.sh [hunk 1]
```diff
@@ -23,41 +23,22 @@ $BRAKE import:brca fname="$(echo "$x" | sed -e 's:.*pseudonymised_data/\(.*\):\1
 done
 }
 
-RTD () {
-echo $DIRPATH/$FILEPATH 
+RTD () {
```
Comment (ollietulloch) [inline lib/import/brca/scripts/bash/Import_all_brca_interactive.sh:26]: Is this just additional whitespace?

**Review by ollietulloch — CHANGES_REQUESTED** [review]
Thanks @shilpigoeldev, looks good.

Just some additional whitespace in the bash script.

Also are the `_deprecated.rb` versions on the handler and persister still needed?

File: lib/import/brca/scripts/bash/Import_all_brca_interactive.sh (hunk 1 above)
Comment (shilpigoeldev) [reply lib/import/brca/scripts/bash/Import_all_brca_interactive.sh]: Hi Ollie, I have removed it now.

**Comment by shilpigoeldev** [comment]
I have removed the extra space and extra files for Newcastle in repo. Have re-run the counts and tests to ensure no impact of removing the deprecated files. Thank you Ollie 🙏

**Review by ollietulloch — APPROVED** [review]
Thanks for the updates

**Comment by shilpigoeldev** [comment]
Thanks Ollie, its all done now and merged inn main develop branch !!

**Pull request merged**